GameData/RealismOverhaul/RO_SuggestedMods/Bluedog_DB/RO_BDB_Pioneer_NEW.cfg

```diff
@@ -0,0 +1,453 @@
+@PART[bluedog_Pioneer_ProbeCore]:FOR[RealismOverhaul]
+{
+	%RSSROConfig = True
+	%rescaleFactor = 1.6
+	@mass = 0.052
+	@maxTemp = 773.15
+	@title = Pioneer 10/11 Probe Core
+	@manufacturer = TRW
+	@description = Breakthroughs in materials and manufacturing have allowed engineers to design a super lightweight hexagonal satellite bus for cheap that can take exploration to new heights. Modular in design, this core is designed with slots to accommodate various expansions like control systems, electrical systems, thermal control systems, experiment bays, and more. <#ef7b06> AUTHOR'S NOTE: Add the Pioneer 10/11 Stellar Reference Assembly to your craft to gain additional SAS modes.</color>
+	%skinMaxTemp = 873.15
+	%crashTolerance = 12
+	%breakingForce = 250
+	%breakingTorque = 250

```

Comment:
Since you duplicate these settings [plus maxTemp] in every part, can you extract them to a single patch at the top?
You can also perhaps just remove these lines except for maxTemp/skinMaxTemp, since they match the RO defaults configured in <https://github.com/KSP-RO/RealismOverhaul/blob/master/GameData/RealismOverhaul/RealismOverhaul_Global_Config.cfg#L91-L100>

You may or may not want to leave the maxTemp/skinMaxTemp lines.  Those differ from the global defaults, but are actually harsher.  (773/873 instead of 1073/1073).

```
@PART[bluedog_Pioneer_ProbeCore,bluedog_Pioneer_HGA,bluedog_Pioneer_SNAP19,...]:FOR[RealismOverhaul]
{ ... }
```
This will make many patches just become:
```
@PART[bluedog_Pioneer_blah]:FOR[RealismOverhaul]
{
	%RSSROConfig = True
	%rescaleFactor = 1.6
	@mass = <something>
	@title = <something>
	@manufacturer = <something>
	@description = <something>
}
```